src/ORM/Table.php

```diff
@@ -96,8 +96,10 @@
  * - `buildValidator(EventInterface $event, Validator $validator, string $name)`
  *   Allows listeners to modify validation rules for the provided named validator.
  *
- * - `buildRules(EventInterface $event, RulesChecker $rules)`
- *   Allows listeners to modify the rules checker by adding more rules.
+ * - `buildRules(RulesChecker $rules)`
+ *   Allows table classes to modify the rules checker by adding more rules. The
+ *   `Model.buildRules` event will also include an `$event` parameter before the 
```

Comment:
This whole docblock section is confusing. Why are we mixing callbacks and events together?

The comment about `Model.buildRules` is confusing as it doesn't explain the relationship between Table::buildRules and Model.buildRules.